pyvcloud/vapp.py

```diff
@@ -929,6 +929,7 @@ def get_vms_details(self):
         if children:
             vms = children.get_Vm()
             for vm in vms:
+                id = vm.get_id()
```

Comment:
`id` is reserved for the built-in `id` function. please use a different name

Reply:
Oops, sorry. I'll update the request.